dev-util/gperf/files/gperf-3.1-clang-16-wregister.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/getline.cc b/lib/getline.cc
```

Comment:
1. Add a link to the commit which it's from (ideally include the full metadata from it, get it from .patch on some site, then paste it in)
2. Run iwdevtools `scrub-patch` on this
3. Include a link at the top too for the relevant Gentoo bug

Reply:
done